.github/workflows/schedule.py

```diff
@@ -171,55 +172,47 @@ def create_issue(self, title_date: date) -> None:
         if issues:
             log.info('At least one matching issue already exists: %r', issues)
         else:
+            repository = 'DataBiosphere/azul' if repository is None else repository
+            type = self.properties.get('type')
+            assert type is not None, type
```

Comment:
No point in printing the value since it can only be possibly be `None` if the assertion fired
```suggestion
            assert type is not None
```